ds/ss.go

```diff
@@ -288,7 +288,7 @@ func (s *SS) GetGame(id string) (*Game, error) {
 	if err == nil {
 		ret.Players = p
 	}
-	if ret.Overview == "" && ret.ReleaseDate == "" && ret.Developer == "" && ret.Publisher == "" && ret.Images[ImgBoxart] == "" && ret.Images[ImgScreen] == "" {
+	if ret.Overview == "" || ret.ReleaseDate == "" || ret.Developer == "" || ret.Publisher == "" || ret.Images[ImgBoxart] == "" || ret.Images[ImgScreen] == "" {
```

Comment:
If you just want to reject an empty overview you'd don't want to have all or's here that would reject it is anything has an empty value.

`ret.Overview == "" || (ret.ReleaseDate == "" && ret.Developer == "" && ret.Publisher == "" && ret.Images[ImgBoxart] == "" && ret.Images[ImgScreen] == "")`

or something like that would probably be better.